docs/introduction-to-smart-contracts.rst

```diff
@@ -332,15 +332,15 @@ Storage, Memory and the Stack
 =============================
 
 Each account has a persistent memory area which is called **storage**.
-Storage is a key-value store that maps 256 bit words to 256 bit words.
+Storage is a key-value store that maps 256-bit words to 256-bit words.
```

Comment:
Are you sure about the `-`? Wouldn't you say "this is a 200 l tank" or "a 2 m ladder"?